commands/Music/volume.js

```diff
@@ -1,32 +1,39 @@
-const { Command } = require("klasa");
+const MusicCommand = require("../../lib/structures/MusicCommand");
 
-module.exports = class extends Command {
+module.exports = class extends MusicCommand {
 
     constructor(...args) {
         super(...args, {
+            requireMusic: true,
             runIn: ["text"],
             cooldown: 8,
             aliases: ["changevol", "setvolume"],
-            permissionLevel: 0,
             requiredPermissions: ["USE_EXTERNAL_EMOJIS"],
             description: msg => msg.language.get("COMMAND_VOLUME_DESCRIPTION"),
             extendedHelp: "No extended help available.",
             usage: "[volume:integer]"
         });
-        this.music = true;
     }
 
     async run(msg, [volume]) {
         if (!volume) return msg.sendMessage(`🔈 | ***Guild's Current Music Volume is:*** ${msg.guild.configs.musicVolume}`);
         if (!msg.hasAtLeastPermissionLevel(3)) return msg.reply("<:penguError:435712890884849664> You are not a **Pengu DJ** to change the volume.");
         if (volume <= 100 || volume >= 0) {
```

Comment:
Use klasa's arg min max, max 150 btw

Reply:
I am actually going to cap it at 100 because 150 doesn't do well with all songs and it's easier for people to understand and use anyways.